src/pages/CommunityPage.vue

```diff
@@ -45,7 +51,11 @@ export default defineComponent({
     ];
     const city = ref<string>('');
 
+    const listCardsFollowImport = listCardsFollow as unknown;
```

Comment:
You can use Interfaces to Define JSON Structures in TypeScript instead of unknown type.